src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalPersons.getTypicalEduMate;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code SortCommand}.
+ */
+public class SortCommandTest {
```

Comment:
@seanfirefox Feel free to add test cases if you'd like. Otherwise, this should be enough.